packages/types/src/search.ts

```
@@ -0,0 +1,110 @@
+  | UserSearchResult 
+  | TagSearchResult 
+  | ActionSearchResult;
+
+// Search filters
+export interface SearchFilters {
+  tags?: string[];
+  minRating?: number;
+  maxRating?: number;
+  dateRange?: {
+    start: string; // ISO date string
+    end: string;   // ISO date string
```

ESLint

[error] Delete `··`

(prettier/prettier)